### PR TITLE
Pass `false` instead of `null` for admin enforcement

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Although the GitHub API description states we should be passing `nil` to
the field when we want it to be unset, the API does not respond
correctly and we must instead explicitly pass `false`.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @cjwagner @fejta 

I have a GitHub support ticket open but for now ...

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -207,7 +207,6 @@ new_git_repository(
     name = "operator_framework_community_operators",
     commit = "42131df7167ec0b264c892c1f3c49ba9a72142da",
     remote = "https://github.com/operator-framework/community-operators.git",
-    shallow_since = "1559569397 +0200",
```
Comment (cblecker) [inline WORKSPACE]: Is this change intended? Seems unrelated.

Comment (stevekuznetsov) [reply WORKSPACE]: I will open a separate PR for it but it needs to happen anyway since this is blocking all dev work using `bazel` ...
